profiles/base/cluster-role.yaml

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
Same here
Profiles are not namespace-scoped, we don't need to aggregate them.

Reply:
I'll make this change.